### PR TITLE
Remove labels from network policies

### PR DESCRIPTION
**Pull request by jaxels10**
**Description of your changes:**
We do not want our users to deal with labels in the network policies as this subject is already complicated enough as is. They now have to deal with release name and deployment name instead.

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

**Comment by jaxels10** [comment]
I would like some help with this PR from @MadsHT to do the labels correctly. Right now they are mainly added as a new line and not under the _helpers functions.
I am also in doubt whether or not the ingress ports need to be a list, but lets discuss

**Comment by jaxels10** [comment]
This was tested with 2 pods from different deployments. First I tried to curl from one pod to the other without any network policies and it was denied:
image
Then, I tried adding an allowed connection through ingress and egress rules:
image
image
This solved the issue:
image

File: charts/tenant-base/docs/deployment.md [hunk 1]
```diff
@@ -115,18 +115,18 @@ deployments:
...
-            app.kubernetes.io/name: foo
+        - releaseName: dlr
+          deploymentName: data
           ports:
             - protocol: TCP
               port: 8080
 ```
 
-This would allow traffic to flow from pods with the label `app.kubernetes.io/name: foo` to the pods created by the deployment on the port defined with `podPortName: http` in `ports`.
+This allows incoming traffic(ingress) to the pod from pods in the dlr helm release that is under the dlr-data deployment. This is done on port http, which is defined under `ports`.
```
Comment (MadsHT) [inline charts/tenant-base/docs/deployment.md:130]:
```suggestion
This allows incoming traffic(ingress) to the pod from pods in the dlr helm release that is under the 'data' deployment. This is done on port http, which is defined under `ports`.
```

File: charts/tenant-base/docs/deployment.md [hunk 2]
```diff
@@ -115,18 +115,18 @@ deployments:
...
 
-It would also allow traffic to flow from the pods creatd by this deployment to pods with the label `app.kubernetes.io/name: foo` on port `8080/TCP`.
+This also allows outgoing traffic(egress) from the pod to pods in the same dkr-data deployment under the dlr helm release on port `8080/TCP`.
```
Comment (MadsHT) [inline charts/tenant-base/docs/deployment.md:132]:
```suggestion
This also allows outgoing traffic(egress) from the pod to pods in the same 'data' deployment under the dlr helm release on port `8080/TCP`.
```

**Review by MadsHT — CHANGES_REQUESTED** [review]
Semantics, otherwise looks good 👍

File: charts/tenant-base/docs/deployment.md (hunk 1 above)
Comment (jaxels10) [reply charts/tenant-base/docs/deployment.md]: Changed

File: charts/tenant-base/docs/deployment.md (hunk 2 above)
Comment (jaxels10) [reply charts/tenant-base/docs/deployment.md]: Changed

**Review by MadsHT — APPROVED** [review]
LGTM

**Pull request merged**